src/status_im/contexts/wallet/share_community_channel/style.cljs

```diff
@@ -0,0 +1,33 @@
+(ns status-im.contexts.wallet.share-community-channel.style
+  (:require [quo.foundations.colors :as colors]))
+
+(def header-container
+  {:padding-horizontal 20
+   :padding-vertical   12})
+
+(def scan-notice
+  {:color        colors/neutral-40
+   :margin-top   12
+   :margin-left  :auto
+   :margin-right :auto})
+
+(def qr-size 500)
+
+(def qr-code-wrapper
+  {:padding-horizontal 20
+   :align-items        :center})
+
+(defn qr-code-size
+  [total-width]
```

Comment:
I personally would do this like

`(def gradient-cover-padding 20)`
`(def qr-code-padding 12)`


```
(defn qr-code-size
  [total-width]
  (- total-width (* gradient-cover-padding 2) (* qr-code-padding 2) )

(defn gradient-cover-size
  [total-width]
  (- total-width (* gradient-cover-padding 2))
 ``` 
 so it's more explicit of where those values are coming from